### PR TITLE
it for mwe part of #1127

### PR DESCRIPTION
**Pull request by cdietrich**
it for mwe part of #1127

@laeubi  unfortunately the same test that fails locally does not fail in test

### CODE REVIEW

**Comment by laeubi** [comment]
I set this ready for review so we execute the integration tests...

File: tycho-its/src/test/java/org/eclipse/tycho/test/compiler/MavenCompilerPluginTest.java [hunk 1]
```diff
@@ -40,6 +40,15 @@ public void testAdditionalBundles() throws Exception {
         verifier.verifyErrorFreeLog();
     }
 
+
+    @Test
+    public void testAdditionalBundles2() throws Exception {
+        Verifier verifier = getVerifier("compiler.additional.bundles2", true);
```
Comment (laeubi) [inline tycho-its/src/test/java/org/eclipse/tycho/test/compiler/MavenCompilerPluginTest.java:46]:
```suggestion
        Verifier verifier = getVerifier("compiler.additional.bundles2", false, false);
```

Should be more suitable here, then you automatically ignore local artifacts.

**Comment by cdietrich** [comment]
you can find the code i copy & pasted over here:
https://github.com/cdietrich/tycho1127_addjars_issue/tree/cd_mimicMweProblem
i basically just ported the groupid.
i dont know what is different in test

**Comment by cdietrich** [comment]
ok, if i call install goal, it now fails

**Comment by laeubi** [comment]
@cdietrich is the failure of testAdditionalBundles2 (org.eclipse.tycho.test.compiler.MavenCompilerPluginTest) with error:

https://github.com/eclipse/tycho/runs/7307088143

what you would expect here? If yes can you remove the 

`verifier.addCliOption("-Dtycho.resolver.classic=false -Dtycho.localArtifacts=ignore");`

so we can see that it succeeds on the current build? If yes we can merge this ...

**Comment by cdietrich** [comment]
@laeubi yes, updated the pr. it now should become green again

**Comment by laeubi** [comment]
Thanks for the additional test!

**Pull request merged**